src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -369,7 +369,7 @@ def create_managed_disk(cmd, resource_group_name, disk_name, location=None,  # p
     if security_data_uri:
         option = getattr(DiskCreateOption, 'import_secure')
     elif source_blob_uri:
-        option = getattr(DiskCreateOption, 'import')
+        option = getattr(DiskCreateOption, 'import_enum')
```

Comment:
The attribute name has changed in the new sdk
![image](https://github.com/Azure/azure-cli/assets/37168047/1fe3a275-082c-4de3-8a0c-0c16446a17ae)